### DISCUSSION TITLE
Top films by genres for an specific year or decade and other parameters

### DISCUSSION THREAD

**Discussion by alvarovadilloz** [cinemagoer/cinemagoer#483] [Q&A]
How can I use cinemagoer package to get the top 25 horror films according their imdb rating, with more than 25,000 votes, released in the 2000’s decade, getting also the rating and runtime for each? I found "cg.get_top50_movies_by_genres" but it only has parameter genres as I see it. Please help or tell me if it is not possible with this package.

**Comment by alberanid** [comment]
This discussion is no longer relevant due to issue https://github.com/cinemagoer/cinemagoer/issues/557: as of June 2026 Cinemagoer no longer analyzes web pages and can only rely on non-commercial datasets.
